tests/wpt/meta/content-security-policy/unsafe-hashes/script_event_handlers_allowed.html.ini

```diff
@@ -0,0 +1,3 @@
+[script_event_handlers_allowed.html]
+  [Test that the inline event handler is allowed to run]
+    expected: FAIL
```

Comment:
Seems like the only legitimate failure. Probably a logic error in the csp crate, investigating.

Reply:
Yup, found it. I wasn't passing `source` into the correct function. When implementing I set it to `""` to make progress, but forgot to update it to the actual value.